src/metrics.ts

```diff
@@ -214,7 +248,7 @@ export class MetricsProcessor implements MetricsProcessorInterface {
       return;
     }
 
-    if (this.data.size === 0) {
+    if (this.evaluationAnalytics.size === 0) {
```

Comment:
Nothing wrong here, but `=== 0` always irks me for some reason.
```suggestion
    if (!this.evaluationAnalytics.size) {
```

Reply:
I researched this - and I agree.